src/parser/parse_tree_cl.c

```diff
@@ -12910,14 +12910,44 @@ pt_print_insert (PARSER_CONTEXT * parser, PT_NODE * p)
   b = pt_append_varchar (parser, b, r1);
   if (r2)
     {
-      char *class_name = NULL;
-
       b = pt_append_nulstring (parser, b, " (");
 
-      while (class_name = strstr ((char *) r2->bytes, (char *) r1->bytes))
+      if (p->info.insert.hint & PT_HINT_USE_SBR)
 	{
-	  strcpy (class_name, class_name + r1->length + 1);
-	  r2->length = r2->length - r1->length - 1;
+	  char *class_name = NULL;
+	  char *column_list = NULL;
+	  char *pos = NULL;
+
+	  class_name = (char *) r1->bytes;
+	  column_list = (char *) r2->bytes;
+
+#if !defined(NDEBUG)
+	  assert (class_name != NULL && strlen (class_name) != 0 && strlen (class_name) != r1->length);
+	  assert (column_list != NULL && strlen (column_list) != 0 && strlen (column_list) != r2->length);
```

Comment:
rewrite assert
the strlen(class_name) != r->length is correct?
isn't it to check strlen(class_name) == r->length ?

Reply:
@byungwook-kim It is removed.